renesas/FreeRTOS
# Fspra 3430 updated free rtos plus tcp to v4.2.1

Updated FreeRTOS plus TCP submodule to v4.2.1

Description
-----------

Update FreeRTOS plus TCP submodule to v4.2.1

Test Steps
-----------

It is tested basic AWS test suites. (It is testd with echo_server.go and `FreeRTOS/FreeRTOS-Plus/Test/FreeRTOS-Plus-TCP/Integration/Full-TCP-Suite/Test_Code/Test_Cases/test_tcp.c`)

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
